Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.2.json

```diff
@@ -30,586 +30,651 @@
                             "default": {},
                             "required": ["type"],
                             "additionalProperties": true,
-                            "properties": {
-                                "type": {
-                                    "$id": "#markup/type",
-                                    "type": "string",
-                                    "title": "Basic type",
-                                    "enum": ["Fiducial", "Line", "Angle", "Curve", "ClosedCurve", "Plane", "ROI"]
+                          "properties": {
+                            "type": {
+                              "$id": "#markup/type",
+                              "type": "string",
+                              "title": "Basic type",
+                              "enum": [ "Fiducial", "Line", "Angle", "Curve", "ClosedCurve", "Plane", "ROI" ]
```

Comment:
Apologies for nitpicking on this, but looks like an unintended change to indentation here that is causing more line diffs in this PR than there really are.